### DISCUSSION TITLE
Unnecessary dependencies on unbound?

### DISCUSSION THREAD

**Discussion by elobdog** [Homebrew/discussions#382] [Everyday usage] [answered]
When I try to remove unbound, it gives me the following error message:
```
$ brew uninstall unbound
Error: Refusing to uninstall /usr/local/Cellar/unbound/1.13.0
because it is required by claws-mail, gnupg, gnutls, gpgme, msmtp, mutt and sylpheed, which are currently installed.
You can override this and force removal with:
  brew uninstall --ignore-dependencies unbound
```

Since unbound is a recursive DNS resolver, none of the formulae listed should have a hard dependency. Is this something that should be addressed? (unbound anyways won't start, since it does not have privileges to bind to port 53, nor a user called unbound, which is requires to run)
```
[1609185103] unbound[5324:0] fatal error: user 'unbound' does not exist.
[1609185107] unbound[5329:0] error: can't bind socket: Permission denied for ::1 port 53
```

**Comment by carlocab** [comment, marked as answer]
The `gnutls` configure script definitely checks for `unbound`:

    checking for unbound library... yes

so I wouldn't say that this is an entirely useless dependency here. I suspect the case is similar for those other formulae.

**Reply by elobdog** [reply]
You're right -- I do see libunbound mentioned on GnuTLS page: https://gnutls.org/download.html for DNSSEC functionality. Maybe that's the case with remaining software too, though it seems a I can simply remove unbound by ignoring those dependencies and everything works (tells me unbound is optional). But oh well... :-)

**Reply by carlocab** [reply]
Removing unbound might work. Note, however, that gnutls was compiled with an unbound library present, so making that library unavailable at runtime could break it.

Doing `brew upgrade` with any formula that depends on unbound is also going to reinstall unbound.
